lib/kubernetes-deploy/logger.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+require 'logger'
```

Comment:
I know you moved this according to my [suggestion](https://github.com/Shopify/kubernetes-deploy/pull/71#discussion_r108981266), but now that I see it I don't like it very much :P. It seems a bit sketchy to inject a method into the module by loading this file... but making it `KubernetesDeploy::Logging.logger.info` or something is gross too. @kirs  you often have great ideas about gem organization... wdyt we should do here? Background: The other PR moved `require 'kubernetes-deploy/runner'` to the bottom of `lib/kubernetes-deploy.rb` because it requires Runner, which requires Kubectl, which requires the logger (and that seemed like a yellow flag on code organization to me).